### ISSUE TITLE
Not working with F7@6.3.5 & VueJS & SCSS

### ISSUE THREAD

**Issue by erwinyusrizal** [valnub/Framework7-Plugin-Welcomescreen#64]
Hi this plugin is not working with F7 v6.3.5 and VueJS, it shows only blank white screen with "skip" link top right screen, I did all the instructions in the doc include the css import

Cheers

**Comment by erwinyusrizal** [comment]
I think i got the problem, in v6.3.5 the plugin params is not working, so the plugin always use the default params

https://github.com/valnub/Framework7-Plugin-Welcomescreen/blob/0d470be7934ad49e6e9c2a16aa43a0a2c1d1cdbc/index.js#L171

any idea @valnub

**Comment by valnub** [comment]
Hi, thanks for reporting the issue. I tried to replicate the problem and can confirm that with Vue it only shows a blank page with a skip button in the top right corner. I'll have a look at the issue and will come back to you if I can find a fix.

**Comment by erwinyusrizal** [comment]
Any luck @valnub ?

**Comment by valnub** [comment]
So, I looked deeper into the issue and can confirm that F7-Vue seems to ignore the parameters set by f7-welcomescreen plugin. I've opened a ticket with F7 team to verify if this is a bug in F7-Vue or something else is wrong: https://github.com/framework7io/framework7/issues/3952

**Comment by valnub** [comment]
Sorry for the late reply but this one was quite tricky. I fixed the issue in latest version [3.0.10](https://www.npmjs.com/package/f7-welcomescreen).

Also, in your Vue component please add the following code:

```
onMounted(() => {
  f7ready((f7) => {
    f7.welcomescreen.init(welcomescreen_slides, options);
  });
});
```

Let me know if that worked for you :-)

**Comment by erwinyusrizal** [comment]
@valnub sorry just have time to test it. I have tested it last night and yes it's working now.  Thank you so much
